tasks/iptables.yml

```diff
@@ -15,6 +15,12 @@
   apt:  name=ufw state=absent purge=yes
   when: ansible_os_family == 'Debian'
 
+- name: Ensure firewalld is not installed (redhat)
+  yum:
+    name: firewalld
+    state: absent
+  when: ansible_os_family == 'RedHat'
```

Comment:
I always find the ansible "when"s a little difficult to follow. It's a lot easier to see all the tasks relevant to one OS in a separate file. 

I would do it like this, though of course you're welcome to suggest an alternative!

main.yml includes one or more of the following based on OS and distro version:
- debian.yml # for all debian distros
- redhat.yml # for all redhat distros
- redhat7.yml # only when redhat >= 7

Reply:
You are right of course. I will do that.

Reply:
I think I will instead encapsulate the distro-specific tasks into `block`s. They are interleaved with distro-agnostic tasks and it seems weird to `include_tasks` a file with two tasks, jump back into the main file, do two tasks there, then include another file with one task.

I think doing blocks should still make it easier to understand, and as a bonus I found a bug while looking over this.

Reply:
Sounds good!

Reply:
I'm gonna resolve this now, as I put the distro-specific tasks in blocks and managed to get rid of the CentOS 7 block altogether.